src/Components/WindowArea/Window/Window.module.css

```diff
@@ -32,46 +32,43 @@
   background: rgba(3, 21, 41, 0.774);
   color: #fff;
 }
-
 .titlebar {
+  height: 32px;
   display: flex;
   flex-direction: row;
   align-items: center;
   justify-content: space-between;
+  padding: 0px 0 0 10px;
```

Comment:
Can you try decreasing the padding? It's fine but less padding might look better I guess.

Reply:
sure thing! 8px padding seems way better yep